torch/onnx/symbolic_opset12.py

```diff
@@ -9,7 +10,56 @@
 
 # This file exports ONNX ops for opset 12
 
+black_listed_operators = [
```

Comment:
unrelated to this PR; making sure to blacklist updated ops in opset 12 that are not updated in PyTorch yet.
Will submit a separate PR to implement those.

Reply:
Yes, let's do that.